src/main/java/com/blackducksoftware/integration/jira/issue/handler/JiraIssueHandler.java

```diff
@@ -317,7 +317,7 @@ private void updateDefaultIssueProperties(BlackDuckIssueModel blackDuckIssueMode
                 alertCategory = "Vulnerability";
             }
 
-            AlertIssueSearchProperties alertIssueSearchProperties = new AlertIssueSearchProperties("Black Duck", "Project", blackDuckIssueFieldTemplate.getProjectName(), "Project Version",
+            AlertIssueSearchProperties alertIssueSearchProperties = new AlertIssueSearchProperties("Black Duck", blackDuckURL, "Project", blackDuckIssueFieldTemplate.getProjectName(), "Project Version",
```

Comment:
What kind of validation should we do for the Black Duck URL? There may be issues if the user configures "https://hub.server.mine:8443/blackduck" vs "https://hub.server.mine:8443/blackduck/"